DataFormats/PatCandidates/src/PackedCandidate.cc

```diff
@@ -92,6 +68,66 @@ void pat::PackedCandidate::unpack() const {
       p4c.release();
     }
 }
+
+/*void pat::PackedCandidate::legacyCovarianceUnpacking() const {
```

Comment:
is it time to remove it now?
I see a commit https://github.com/cms-sw/cmssw/pull/18092/commits/43d9302def1e555713d017f92c7207e52cf7766d with a name suggestive that the bw compatibility was checked